src/App.js

```diff
@@ -129,11 +67,9 @@ const FilterByDayOfWeek = ({ filterByDay, setFilterByDay }) => {
 
   return (
       <>
-        <hr className='horizontal-line'/>
-        <section className="is-horizontal-scrollable centered">
+        <div class="control">
```

Comment:
Could you revert this change? It's causing the search bar and the list of days to not show up correctly on mobile

https://user-images.githubusercontent.com/13685735/109370462-a7f3da80-785d-11eb-9613-1ead3b4657bf.mov